public/css/service-grid.less

```diff
@@ -0,0 +1,75 @@
+/*! Icinga Web 2 | (c) 2015 Icinga Development Team | GPLv2+ */
```

Comment:
Please remove the license header.

And move the entire file to `public/css/view` (new directory)